src/jobs/database_schema_report.yml

```diff
@@ -0,0 +1,75 @@
+---
+description: >
+  Job to generate a schema report for a postgres database using schemaspy. Runs against postgres in docker
+  An HTML report is generated and is available on the artifacts tab of the circleci job
+executor:
+  name: java_localstack_postgres
```

Comment:
not really understanding still why you need a `localstack` executor here.  Shouldn't this just be java and postgres?